demos/multi_channel/object_detection_demo_yolov3/main.cpp

```diff
@@ -277,7 +270,7 @@ std::map<std::string, YoloParams> GetYoloParams(const std::vector<std::string>&
         }
         anchors = maskedAnchors;
 
-        YoloParams param(num, classes, coords, anchors);
+        YoloParams param{.num = num, .classes = classes, .coords = coords, .anchors = anchors};
```

Comment:
Designated initializers aren't standard C++ (yet, anyway). You'll have to remove the field names.